api/v2/models/bouquets/bouquets_model.py

```diff
@@ -40,8 +38,11 @@ def receive_notification_from_google(self):
 
         # TODO: add functionality to receive calendar notifications for purposes of logging them on the relay log table
```

Comment:
is this something that you will have to work on later on?

Reply:
Yes, thats functionality that will be added later on